src/css/sort_option.css

```diff
@@ -0,0 +1,120 @@
+.search_sort {
+  position: relative;
+  user-select: none;
+  width: 26rem;
+  font-size: 1rem;
+  font-weight: 300;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 60px;
+  white-space: nowrap;
+}
+.search_sortTrigger {
+  position: relative;
+  display: grid;
+  height: 100%;
+  align-items: center;
+  padding: 0 22px;
+  color: var(--main-text-color);
+  background-color: var(--main-border-color);
+  cursor: pointer;
+}
+.search_sortTrigger::before {
+  content: '\21F5';
+  font-size: 1.5rem;
+  position: absolute;
+  padding-left: 10px;
+  margin-bottom: 5px;
+}
+@media screen and (max-width: 1024px) {
+  .search_sortTrigger {
+    width: 6ch;
+  }
+  .search_sortOptions {
+    width: max-content;
+  }
+  .search_sortLabel {
+    margin-left: 5px;
+    overflow: hidden;
+  }
+}
+.search_sortOptions {
+  position: absolute;
+  display: block;
+  top: 100%;
+  left: -1px;
+  right: 0;
+  border: 1px solid var(--main-border-color);
+  border-top: 0;
+  background: var(--main-bg-color);
+  transition: all 0.5s;
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+  z-index: 2;
+}
+.search_sort.open .search_sortOptions {
+  opacity: 1;
+  visibility: visible;
+  pointer-events: all;
+}
+.search_sortOption {
+  position: relative;
+  display: block;
+  padding: 0 22px 0 22px;
+  color: var(--main-text-color);
+  line-height: 60px;
+  cursor: pointer;
+  transition: all 0.5s;
+  border-top: 1px solid var(--main-border-color);
```

Comment:
This, combined with

```css
border: 1px solid var(--main-border-color);
border-top: 0;
```

on the parent, adds the borders for the drop-down options.